src/pages/myTodo/tabs/ScrapPage.tsx

```
@@ -21,17 +19,14 @@ const dummyJobs = Array.from({ length: 9 }, (_, i) => ({
-
   const [jobSort, setJobSort] = useState('최신순');
   const [eduSort, setEduSort] = useState('최신순');
-
   const [jobRegion, setJobRegion] = useState('지역 선택');
   const [eduRegion, setEduRegion] = useState('지역 선택');
-
   const [isModalOpen, setIsModalOpen] = useState(false);
   const [jobPage, setJobPage] = useState(1);
   const [eduPage, setEduPage] = useState(1);
-  const [selectedCardId, setSelectedCardId] = useState<string | null>(null);
+  const [selectedCardId, _setSelectedCardId] = useState<string | null>(null);
```

ESLint

[error] '_setSelectedCardId' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)